### DISCUSSION TITLE
How to translate the tooltip text in candlestick chart?

### DISCUSSION THREAD

**Discussion by Gabsato02** [apexcharts/apexcharts.js#3230] [Q&A]
Hi!

I'm using Apexcharts to build a candlestick chart, but I need the tooltip to be in pt-BR instead of english. How can I translate/replace that?

<img width="287" alt="Captura de Tela 2022-06-23 às 17 18 47" src="https://user-images.githubusercontent.com/67379143/175392146-25f46095-9f2c-4585-a5bb-288359bf553f.png">

**Comment by Tenacius-p** [comment]
Using tooltip.custom worked for me

https://apexcharts.com/docs/options/tooltip/#custom
